canonical-web-and-design/maas-ui
# fix(ui): fix double tar of logs

## Done

- Update the API call to return the logs without the tar option.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Point your UI to bolla.
- Go to /MAAS/r/machine/mbfgbq/logs/events
- Open the download menu and download the curtin log
- Open the tar file and it should extract the contents, not another .tar file.

## Fixes

Fixes: #2498.
